vuepress/docs/docs/federation.md

```diff
@@ -0,0 +1,81 @@
+# Federation
+
+**Federation** is an optional module which can be included in your configuration to enroll with a federated schema.
+
+## Dependencies
+
+`caliban-federation` only depends on `caliban-core` and is very unobtrusive.
+
+To use, add the following line to your `build.sbt` file:
+
+```
+libraryDependencies += "com.github.ghostdogpr" %% "caliban-federation" % "0.7.3"
```

Comment:
Can you make it `0.7.4`? So that next time I release, a find & replace will find it 😄